.travis.yml

```diff
@@ -64,6 +65,7 @@ install:
 script:
   - composer analyze
   - if [[ $PHPDBG != 1 ]]; then xdebug-enable; fi
+  - if [[ $PHPDBG != 1 ]]; then $PHPUNIT_E2E; else phpdbg -qrr $PHPUNIT_E2E; fi
```

Comment:
I'm probably missing something here, but how is coverage generated from this, when it is ran separately from the other phpunit tests and without --coverage options

Reply:
That’s correct. Is there a way to join coverage from this run and the next? Is that going to make any sense at all? 

If not, I’ll just remove this and another line. We can have them both in one suite for some reason. It takes way over ten minutes to run on Travis.

Reply:
Maybe https://github.com/sebastianbergmann/phpcov may work

Reply:
Well, it appears I better spend time making actual tests work than making phpcov show kind-of total coverage where it can't be used by later Infection run anyway.